src/js/records/operations/issuance.record.js

```diff
@@ -22,7 +30,7 @@ export class IssuanceRecord extends OpRecord {
   }
 
   get isIncoming () {
-    return true
+    return this.accountId === this.counterparty
```

Comment:
This row doesn’t work as expected. @LemonJenkins gonna investigate the problem a bit deeper and fix `isIncoming` for all the records.

Reply:
ok